contrib/libxls/utils/utils_libxls.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Google LLC
```

Comment:
The API is ugly.
As this still requires LibxlsSapiSandbox to outlive LibXls, one could without sacrificing any safety:
```
class XlsWorkbook {
public:
  static absl::StatusOr<XlsWorkbook> Open(std::string& filename, const std::string& encode);
  absl::StatusOr<XlsSheet> OpenSheet(uint32_t index);
};

struct XlsCell {
  int type;
  std::variant<double, bool, std::string> value;
}

class XlsSheet {
public:
  size_t GetRowCount() const;
  size_t GetColCount() const;
  absl::StatusOr<XlsCell> GetCell(uint32_t row, uint32_t col);
};
```
Modulo a LibxlsSapiSandbox* param for XlsWorkbook::Open if opening multiple workbooks using same sandbox or more control over sandbox's config is desired.